packages/material/scss/button/_layout.scss

```diff
@@ -1,36 +1,41 @@
-$bare-button-hover-opacity: .12 !default;
-$outline-button-border-width: 2px;
+$flat-button-hover-opacity: .12 !default;
+$flat-button-focused-opacity: .12 !default;
+$flat-button-active-opacity: .16 !default;
+$flat-button-selected-opacity: .16 !default;
+$outline-button-border-width: 2px !default;
+
 @import "~@progress/kendo-theme-default/scss/button/layout";
 
 @include exports("button/layout/material") {
-    .k-button.k-outline {
+
+    .k-button {
         &::before {
-            content: "";
             display: block;
-            opacity: 0;
-            pointer-events: none;
-            position: absolute;
-            z-index: 0;
-            top: -$button-border-width;
-            left: -$button-border-width;
-            right: -$button-border-width;
-            bottom: -$button-border-width;
+            transition: opacity .4s cubic-bezier(.25, .8, .25, 1);
         }
     }
 
-    .k-button-group {
-        &.k-button-group-outline {
-            .k-button {
-                border-width: $outline-button-border-width;
-                border-style: solid;
-                box-sizing: border-box;
-                padding-top: $button-padding-y / 2;
-                padding-bottom: $button-padding-y / 2;
-            }
 
-            .k-button ~ .k-button {
-                margin-left: -$outline-button-border-width;
-            }
+    // Outline button
+    .k-button.k-outline {
+        padding-top: calc( #{$button-padding-y} - #{$outline-button-border-width} );
+        padding-bottom: calc( #{$button-padding-y} - #{$outline-button-border-width} );
+        padding-left: calc( #{$button-padding-x} - #{$outline-button-border-width} );
+        padding-right: calc( #{$button-padding-x} - #{$outline-button-border-width} );
+        border: $outline-button-border-width solid currentColor;
```

Comment:
* Assuming we always want the outline border to behave as such.
* use calculated padding values

Reply:
That's great, and I believe it fixes a a bug in the calculation (`$button-padding-y / 2` was previously expected to equal `$outline-button-border-width`, if I understand correctly).

I suggest using `add-two()`, as it's cleaner and may remove the `calc()` expression.

Reply:
Prior adding postcss-calc as dependency, I would use add-two.

Now that we have it, we can safely rely, as it will optimize the expression if the values are using the same units.

Reply:
I suggest using `add-two` because of the less symbols involved. Plus, the themes may be compiled without postcss-calc in customer applications (angular-cli, react-cli)